src/main/java/uk/gov/hmcts/reform/sscs/service/AddNoteService.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.sscs.service;
+
+import static java.util.Objects.nonNull;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.ccd.domain.Note;
+import uk.gov.hmcts.reform.sscs.ccd.domain.NoteDetails;
+import uk.gov.hmcts.reform.sscs.ccd.domain.NotePad;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+
+@Service
+public class AddNoteService {
+
+    protected final UserDetailsService userDetailsService;
+
+    @Autowired
+    public AddNoteService(UserDetailsService userDetailsService) {
+        this.userDetailsService = userDetailsService;
+    }
+
+    public void addNote(String userAuthorisation, SscsCaseData sscsCaseData, String note) {
+        if (nonNull(note) && StringUtils.isNoneBlank(note)) {
```

Comment:
nice to have: It would read better if isNotEmpty is used instead to these two checks.